client/src/featureform/register.py

```diff
@@ -3424,15 +3453,20 @@ def list_training_sets(self, local=False):
             return list_local("training-set", [ColumnName.NAME, ColumnName.VARIANT, ColumnName.STATUS])
         return list_name_variant_status_desc(self._stub, "training-set")
 
-    def list_models(self, local=False):
+    def list_models(self, local=False) -> List[Model]:
         """List all models. Prints a list of all models.
 
         Returns:
             models (List[Model]): List of Model Objects
         """
+        models = []
         if local:
-            return list_local("model", [ColumnName.NAME, ColumnName.STATUS, ColumnName.DESCRIPTION])
-        return list_name_status_desc(self._stub, "model")
+            rows = list_local("model", [ColumnName.NAME, ColumnName.STATUS])
```

Comment:
Models shouldn't have a status

Reply:
A model's status didn't make much sense to me either; however, several other record types has a status field (e.g. entity and user) and the DDL for the `models` table already contained it so I wasn't certain whether I should remove it or not.

I'll remove it.

Reply:
The status for user and entity should probably be removed as well. I would remove it from models and leave the others for now just incase other things (like prints) need to be modified so that those arent broken (out of scope)